src/System.Net.WebClient/tests/WebClientTest.cs

```diff
@@ -18,7 +18,7 @@ namespace System.Net.Tests
     public class WebClientTest
     {
         [Fact]
-        public static void DefaultCtor_PropertiesReturnExpectedValues()
+        public static void WebClient_DefaultCtor_PropertiesReturnExpectedValues()
```

Comment:
Because we have tests, in System.Net.Http, for example, where HttpClientHandlerTest.cs doesn't use class name in the test method name:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

If the CI system is having a problem reporting the full namespace/class/method for the tests, we should get that fixed.  cc: @safern